src/components/MentorsList/MentorsList.js

```diff
@@ -20,24 +22,50 @@ export default class MentorsList extends Component {
   };
 
   componentWillReceiveProps(newProps) {
```

Comment:
Should this be renamed to `UNSAFE_componentWillReceiveProps`?

Reply:
Yes, it is already deprecated and will became obsolete in the near future, we should refactor this to a functional component

Reply:
@ramirozap lol, I was just about to create an issue for this, but you beat me there by 30 seconds.

I think we can launch this safely as is, and circle back on the refactor after the launch.  Do you feel the same?

Reply:
Yes, is not a major issue, yet, still works as expected, but we should do it before updating react to a major version, just in case.

Reply:
I feel the same :)